changelogs/current.yaml

```diff
@@ -204,6 +204,10 @@ bug_fixes:
 - area: docker/publishing
   change: |
     Update base images to resolve various glibc vulnerabilities.
+        Fixed a bug where redis key with ``%`` in the key is failing with a validation error.
```

Comment:
Rather than a bug, I personally think this is a new feature or minor behavior change. Before this patch, when we emitting grpc logs, only downstream filter state will be used. And after the patch, both downstream and upstream filter state may be tried.

Reply:
yeah, bad merge.